src/Controller/ElasticsearchController.php

```diff
@@ -7,45 +7,35 @@
 use Elasticsearch\Common\Exceptions\ElasticsearchException;
 use Elasticsearch\Common\Exceptions\Missing404Exception;
 use Elasticsearch\Common\Exceptions\NoNodesAvailableException;
+use EMS\CommonBundle\Elasticsearch\Response\Response as CommonResponse;
```

Comment:
Don't like its called CommonResponse, Its a ElasticsearchResponse

Reply:
Me neither but there is another use Symfony\Component\HttpFoundation\Response;